assets/images/forest-watcher/forest-watcher-user-guide-en/desktop/test

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Is this file necessary, Karla?

I'm going to merge so that the user guide is linked, but maybe this file can be removed in a separate pull request.

Reply:
No it's not - I always have to create a test file because I can't figure out how to create a folder with nothing in it. Maybe you can show me sometime?

Reply:
So normally what's done to create a folder that will be empty is to add a `.gitkeep` text file with no content. So basically the same thing that you are doing but with a different name that makes its purpose more clear.